### DISCUSSION TITLE
Grid Row Number

### DISCUSSION THREAD

**Discussion by rodpinst** [vikramlearning/blazorbootstrap#600] [Q&A] [answered]
Hi all, Fisrt of all I would like to congrats @gvreddy04 by the excellent work with blazorbootstrap. We are using in some projects and it works weel most of time, and it integrated pretty well with an bootstrap layout the we acquired.

Well I would like to know if there a way to show, in grid componente, row number?

**Comment by gvreddy04** [comment, marked as answer]
> Hi all, Fisrt of all I would like to congrats @gvreddy04 by the excellent work with blazorbootstrap. We are using in some projects and it works weel most of time, and it integrated pretty well with an bootstrap layout the we acquired.

@rodpinst  Thank you for your feedback.

> Well I would like to know if there a way to show, in grid componente, row number?

Currently not supported. We will consider incorporating it if adding this feature provides value.

I have a question: When paging is enabled with a page size of 10, what is your expectation for row numbering on the second page? Should it start from 11 or continue from 1?

**Reply by rodpinst** [reply]
Thank you for the answer.

Well for me the row number in page 2 should consider registers from previous page, so should start in 11.
